scripts/backtest_uk_gsp.py

```diff
@@ -144,7 +140,7 @@ def get_available_t0_times(start_datetime, end_datetime, config_path):
     # Pop out the config file
     config = datapipes_dict.pop("config")
 
-    # We are going to abuse the `create_t0_and_loc_datapipes()` function to find the init-times in
+    # We are going to abuse the `create_datapipes()` function to find the init-times in
```

Comment:
genuine questions, is "abuse" a technical term here?

Reply:
or does it mean my interpretation which is "use in a hacky way"

Reply:
No its not. But we are using this function in a really dirty way. We should really be writing a new clean function which does exactly what we want and refactoring create_datapipes rather than what we do here